### PR TITLE
Settings with drop down menu

### PR DESCRIPTION
**Pull request by LowriKate**

### CODE REVIEW

File: components/ToggleSwitch.tsx [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+  &:before {
+    content: "";
+    position: absolute;
+    top: 4px;
+    left: 4px;
+    width: 32px;
+    height: 32px;
+    background: #fff;
+    border-radius: 90px;
```
[the remark this answers is not in the document]
Comment (silverlight513) [reply components/ToggleSwitch.tsx:28]: If you're looking to make a cricle or something, then you can use 50% as the value

File: .env [hunk 2]
```diff
@@ -0,0 +1,2 @@
+# this should always be in caps
+DB_URL=mongodb+srv://Scoop:Moronorenacoch@clustersofscoop.0olrge8.mongodb.net/?retryWrites=true&w=majority
```
Comment (silverlight513) [inline .env:2]: .env files should not be committed. Can you add this file to the `.gitignore` file please (it's just a list of file and folder paths relative to its location

**Review by silverlight513 — CHANGES_REQUESTED** [review]

File: components/Input.tsx [hunk 3]
```diff
@@ -9,18 +9,19 @@ const Input = styled.input`
   background-color: white;
   outline: #c4c4c4;
   align-items: center;
-  font-size: 1.25rem;
+  font-size: 1rem;
```
[the remark this answers is not in the document]
Comment (LowriKate) [reply components/Input.tsx:12]: The font becomes much smaller when I remove this.

File: components/PageHeader/DropDownContainer.tsx [hunk 4]
```diff
@@ -0,0 +1,5 @@
+import styled from "styled-components";
+
+const DropDownContainer = styled("div")``;
```
[the remark this answers is not in the document]
Comment (LowriKate) [reply components/PageHeader/DropDownContainer.tsx:3]: Deleted, don't think I was using this anyway.

**Review by silverlight513 — APPROVED** [review]

**Pull request merged**